test/test_metadata_flattener.py

```diff
@@ -66,39 +117,13 @@ def test_flatten_with_dirname(self):
 
         flattener = Flatten()
 
-        flattener.add_bundle_files_to_row([self.doc1, self.doc2], dir_name="1234-aaaa")
+        flattener.add_bundle_files_to_row(self.manifest, self.test_docs, dir_name="1234-aaaa")
         expected = os.path.join(BASE_PATH, 'test2.csv')
         output = os.path.join(BASE_PATH, 'test2_tmp.csv')
         flattener.dump(filename=output)
         self.assertTrue(filecmp.cmp(expected, output))
         os.remove(output)
 
-
```

Comment:
I removed this test since the default filter no longer replaces the file-type with `fastq.gz` or any other file type.